### PR TITLE
Tutorial on account, job, backend info

### PR DESCRIPTION
**Pull request by stephanielee9**

### CODE REVIEW

File: docs/source/get_started/access_info/access_info_css.ipynb [hunk 1]
```diff
@@ -0,0 +1,300 @@
+{
```
Comment (bharat-thotakura) [inline docs/source/get_started/access_info/access_info_css.ipynb:1]: <p>Should we uniformly use:</p><pre class="ql-syntax" spellcheck="false">%pip install -q cirq-superstaq
</pre><p>across all the notebooks for a cleaner output for the user?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (bharat-thotakura) [inline docs/source/get_started/access_info/access_info_css.ipynb:1]: <p>I think the link for 'this guide' is broken</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (bharat-thotakura) [inline docs/source/get_started/access_info/access_info_css.ipynb:1]: <p>To match the qss version, I think there should be a markdown cell for: </p><p> </p><p>"If are interested in increasing your balance or have more information on your user role, please reach out to us at <a href="mailto:superstaq@infleqtion.com" target="_blank">superstaq@infleqtion.com</a> or join our <a href="https://join.slack.com/t/superstaq/shared_invite/zt-1wr6eok5j-fMwB7dPEWGG~5S474xGhxw" target="_blank">Slack workspace</a>."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (bharat-thotakura) [inline docs/source/get_started/access_info/access_info_css.ipynb:1]: <p>Should it be: "account information, the <code>Service()</code> object" ?</p><p> </p><p>Also, the qss version breaks this up into three bullet points:</p><p> </p><ul><li><code>backends()</code>: Retrieves a list of available backends</li><li><code>get_backend("&lt;backend_name&gt;")</code>: Select your target backend, where&nbsp;<code>&lt;backend_name&gt;</code>&nbsp;is the name of the desired backend</li><li><code>get_backend("&lt;backend_name&gt;").target_info()</code>: Retrieve information on your selected backend, such as number of qubits, native gate set</li></ul><p> </p><p>which I think would also be helpful here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (bharat-thotakura) [inline docs/source/get_started/access_info/access_info_css.ipynb:1]: <p>nit: Perhaps redundant, but having a #comment (like in the qss version: "# selecting the IBM Lagos device") or a markdown cell before with brief leading text along the lines: "For example, here is the kind of information you'll be able to retrieve for, say, "ibmq_lagos_qpu""</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (bharat-thotakura) [inline docs/source/get_started/access_info/access_info_css.ipynb:1]: <p>nit: SuperstaQ -&gt; Superstaq</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

File: docs/source/get_started/access_info/access_info_qss.ipynb [hunk 2]
```diff
@@ -0,0 +1,307 @@
+{
```
Comment (bharat-thotakura) [inline docs/source/get_started/access_info/access_info_qss.ipynb:1]: <p>Link for "this guide" leads to 404 like in the css version.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (bharat-thotakura) [inline docs/source/get_started/access_info/access_info_qss.ipynb:1]: <p>nit: SuperstaQ -&gt; Superstaq.</p><p> </p><p>(Also, just a note, after PR #468, <code>status()</code> would be able to give the following <code>qiskit.providers.jobstatus</code></p><p>: <code>JobStatus.QUEUED</code>, <code>JobStatus.RUNNING</code>, <code>JobStatus.INITIALIZING</code>, <code>JobStatus.CANCELLED</code>,<code>JobStatus.ERROR</code>, <code>JobStatus.DONE</code>... But maybe this doesn't have to be updated now.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (bharat-thotakura) [inline docs/source/get_started/access_info/access_info_qss.ipynb:1]: <p>Minor curiosity: do we have a standard for the capitalization of #comments? Some notebook comments begin with capital letters while others begin with lowercase.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Comment by bharat-thotakura** [comment]
Just some minor nits, but the notebooks look good to go after that!

File: docs/source/get_started/access_info/access_info_css.ipynb (hunk 1 above)
Comment (stephanielee9) [reply docs/source/get_started/access_info/access_info_css.ipynb:1]: <p>Sounds good - what is the equivalent of <code>get_backend()</code> in Cirq / <code>css</code> ?  </p>

File: docs/source/get_started/access_info/access_info_qss.ipynb (hunk 2 above)
Comment (stephanielee9) [reply docs/source/get_started/access_info/access_info_qss.ipynb:1]: I am not sure - Maybe @richrines1 or @dowusu-antwi would know?

File: docs/source/get_started/access_info/access_info_css.ipynb (hunk 1 above)
Comment (bharat-thotakura) [reply docs/source/get_started/access_info/access_info_css.ipynb:1]: <p>My bad, I forgot that with <code>cirq-superstaq</code> you just do <code>cirq._superstaq.Service().target_info("ibmq_qasm_simulator")</code> for example.   </p>

**Review by bharat-thotakura — APPROVED** [review]
LGTM, but maybe others have any further suggestions

File: docs/source/get_started/access_info/access_info_qss.ipynb (hunk 2 above)
Comment (dowusu-antwi) [reply docs/source/get_started/access_info/access_info_qss.ipynb:1]: Just saw this; we don't have a standard for comments. (Maybe we should)

Comment (richrines1) [reply docs/source/get_started/access_info/access_info_qss.ipynb:1]: i've wondered about this too. also whether/when they should end in periods vs. colons vs. no punctuation..

currently we're all over the place, so i usually just copy whatever the previous comment in the file does. the inconsistency definitely bothers me though lol

Comment (stephanielee9) [reply docs/source/get_started/access_info/access_info_qss.ipynb:1]: Opened this issue so we can keep track of this after June 30: https://github.com/Infleqtion/client-superstaq/issues/560

**Pull request merged**